frontend/src/scenes/insights/filters/ActionFilter/ActionFilterRow/ActionFilterRow.tsx

```diff
@@ -442,7 +442,7 @@ function useMathSelectorOptions(
             options.splice(1, 0, {
                 value: countPerActorMathTypeShown,
                 label: (
-                    <div className="flex items-center gap-2">
+                    <div className="flex items-center gap-2 pr-1">
```

Comment:
This is a bit absurd because the only purpose of this `pr-1` is actually canceling out `-mr-1` on the LemonSelect. 😅 The purpose of that negative padding was making this select-within-select look a bit more natural. I think I also did remove `overflow: hidden` from `LemonButton__content` to make this possible, because I didn't see that it would be a problem with something like those project homepage lists – my bad. Let's remove the negative margin though as pulling it off now seems risky.

Reply:
This is my only note.

Reply:
Ah yeah somehow I didn't notice that. I'll make that change.